README.md

```diff
@@ -1,7 +1,7 @@
 # Cookiecutter PyPackage
 
-![Code Quality Checks](https://github.com/mgancita/cookiecutter-pypackage/workflows/Code%20Quality%20Checks/badge.svg)
```

Comment:
I don't think these changes are right as this would direct the project's README to your version / fork of the project

Reply:
Yes you are right. I actually changed those after opening the pull request. I thought this wouldn't affect the PR. So yes revert those changes in the PR.